lib/crypto.js

```diff
@@ -0,0 +1,14 @@
+const bcrypt = require("bcrypt");
+
+async function encryptPassword(password) {
+  const hash = await bcrypt.hash(password, 10);
+  return hash;
+}
+
+async function decryptPassword(password, hash) {
+  const decryptedPassword = await bcrypt.compare(password, hash);
+  return decryptedPassword;
+}
+
+exports.encryptPassword = encryptPassword;
+exports.decryptPassword = decryptPassword;
```

Comment:
its confussing naming, your script didn't decrypt something it compares the two hashes

```suggestion
async function comparePassword(storedPassword, submittedPassword) {
  const resultComparedPasswords = await bcrypt.compare(storedPassword, submittedPassword);
  return resultComparedPasswords;
}

exports.encryptPassword = encryptPassword;
exports.decryptPassword = comparePassword;
```